### PR TITLE
Fill out more of EnhacnedGameObject's Class members and other missing…

### PR DESCRIPTION
**Pull request by CallocGD**
Hopefully this pull request is better than yesterday's one where I was up at 2AM. Seems having GameObject's previous class members helped out but I am still hunting down the remaining ones that slipped through the cracks somehow thanks to this update.

### CODE REVIEW

**Comment by CallocGD** [comment]
I swear the verification system is screwed up.

File: bindings/2.2074/GeometryDash.bro [hunk 1]
```diff
@@ -4798,16 +4812,22 @@ class EnhancedGameObject : GameObject {
...
     bool m_isNoMultiActivate;
     // property 99
     bool m_isMultiActivate;
-    PAD = android32 0x4, win 0x4, android64 0x8, mac 0x8;
-}
+    bool m_activated;
+    bool m_activatedByPlayer1;
+    bool m_activatedByPlayer2;
+    bool m_hasUniqueCoin;
+    cocos2d::CCColor3B m_unkAnimationColor;
```
Comment (altalk23) [inline bindings/2.2074/GeometryDash.bro:4829]: this is supposed to be ccColor3B not uppercase

Comment (CallocGD) [reply bindings/2.2074/GeometryDash.bro]: Fixed

**Review by altalk23 — COMMENTED** [review]
(last member) I'm not so sure about the validity of this as first member of effectriggerobject is also cccolor3b (and colors dont match)
member tests could be helpful

**Comment by altalk23** [comment]
wow github is weird

**Pull request merged**